### ISSUE TITLE
Way to turn down Lycanites mobs, for have more vanilla mobs than Lycanites ones

### ISSUE THREAD

**Issue by Fireztonez** [Lycanite/LycanitesMobs#396]
Hello,
I'm looking for the best way in a long time, to be able to reduce the total number of Lycanites monsters spawn during the night, to have more chances to hatch Vanilla Mobs and more chance to kill them ...

Every night, it's crazy, there are so many Lycanites monsters, it's hard to find a simple zombie, I do not even talk Enderman !! What I want is to have just some extra mobs, which add a variety and a little extra challenge, but right now it's just too much in my opinion for my modpack.

I've already spotted each of the mobs spawn individually, but, I'm looking for a way to be able to pay the total. In addition, because the spawn weight is in Interger, it is very difficult to reduce everything, keeping spawn chances smaller for some mobs than others. So, I would like to know if there is way with the present options to reduce the total spawn of mobs that spawn at night.

Thank's
Fireztonez

**Comment by Lycanite** [comment]
Hey!

In config/lycanitesmobs/lycanitesmobs-spawning.cfg you can find a setting called "Weight Scale" in the Global Spawning section. This is 1.0 by default but scales all weights that mobs from Lycanites Mobs use with the vanilla creature and monster spawners which are shared with vanilla mobs.

You can set this to 0.1 to reduce all weights down to 10% relative to vanilla spawns, or if you're feeling crazy one day 2.0 to double Lycanites Mobs!

I'll keep this issue open as I've recently rewritten the entire spawning system and I'll need to check to make sure this setting is still working as intended.

**Comment by Fireztonez** [comment]
Oki,
Thank's for that fast reply, I will try this!

**Comment by Lycanite** [comment]
I've started documenting the config which you can find on the site here too: http://lycanitesmobs.com/config
Let me know if anything is unclear so I can improve it too. :)

**Comment by Lycanite** [comment]
Oh and you can use '/lm spawners reload' while in game when editing json files btw.
I replied to #365 for ya.

**Comment by Fireztonez** [comment]
For the help I see it and it's really great, but some times, is really hard to find where is the information we need about a specific configuration or just for find reply of a simple question...

Sincerely, I think it can be helpful to use a real wiki template for your help section, but on any case, I think adding a search engine in your website should be really useful... Lot of time, for find help about one single line, I need to open about 5 or 6 page and find in the page to see if the line I want is there (Possibly I'm just misunderstand the classment)...

And, about the website, can you just make sure to don't use a bright Background with bright text again! Is take to much effort to read, this is not cool...

Thank you again for your help!
Fireztonez

**Comment by Lycanite** [comment]
Yeah I've got a wiki in the works but it'll be a while before it's up and ready to use. I'll try and keep the colors on the website more readable as well.

**Comment by Fireztonez** [comment]
I think the configs files for the spawn weight is little confusing and in to mutch different places... I try to understand the way to control the spawn weight, but stangely, I reduce the spawn weight in lot of files, and now I get like 5 nymph around my house, when I try to sleep, 1 time out of 2 a phantom spawn, while I set ""chance": 0.05,". I do not know if 2 spawn weight under 1 set can increase the chances of spawn?


**Small comment, question of possibly improving your mod _(Do not take it badly)_:**
Sincerely, I think that what is the most annoying with your mod is having to juggle between 290 json files, to obtain the result that I wish. I know for example that it's really not obvious things to change the way to control everything, without cutting completely break compatibility with all that is done and with previous versions... Possibly it would be possible to do a configuration file that would override the spawn options, but is just a suggestion.

**Comment by Lycanite** [comment]
I would like to simplify things but everyone wants something completely different which is why the configs are as advanced as they are right now.

**Comment by Fireztonez** [comment]
Yeah,
I understand your point... Having a big mod like that, doing thing's differently from any-else, is hard to make config simple, without remove any settings!!
